synapse/lib/schemas.py

```diff
@@ -668,9 +668,22 @@
             'pattern': s_version.semverstr,
         },
         'build': {
-            'type' 'object'
+            'type': 'object',
             'properties': {
                 'time': {'type': 'number'},
+                'synapse:verstring': {
+                    'type': 'string',
```

Comment:
Do we need both version and version string? This could be a larger philosophical question which boils down to using semver libs to compare versions rather than just `>=` but i like the simplicity 🤔

Reply:
just version string.